### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by noah-d-whitney** [fsyntax/medusa-plugin-printful#3] [Q&A]
Hey Marco, I'm currently working on a full-stack e-commerce site built with a Medusa server that requires printful integration and came across your Printful+Medusa plugin.

I tried your website contact form but it didn't seem to be working, so I wanted to leave this here to see if you'd be interested in an extra hand to help out with contributions to the plugin. I'll be maintaining this site long-term and really love the plugin and idea behind it.

Kindly let me know if there's and small features or bug fixes I could take a stab at. I would love to contribute!

**Comment by fsyntax** [comment]
Hey @noah-d-whitney, thanks for reaching out! Thank you as well for the hint with the form, really need to fix that asap 😁
Regarding the plugin: I'm glad you like the idea :) There are actually quite a few things that need to be done - I just haven't got to create a proper Roadmap/Plans section in the README (will do that today). 
A full integration is a rather big project and since I'm not able to work fulltime on it - any contributions are welcome!

**Reply by noah-d-whitney** [reply]
Sounds good brother, I'll keep an eye out for the roadmap and see what I can do!
